core/components/atoms/avatarSelection/AvatarSelection.tsx

```diff
@@ -0,0 +1,267 @@
+import * as React from 'react';
```

Comment:
The file 'AvatarSelection' and the directory 'avatarSelection' are siblings with same name and only a case different, please use some better naming for the directory.